### PR TITLE
Added Add/Sub for PrivateKey of Monty256/480 and internal

### PR DESCRIPTION
**Pull request by giarc3**

### CODE REVIEW

File: src/api_480.rs [hunk 1]
```diff
@@ -1240,6 +1257,34 @@ pub(crate) mod test {
         assert_eq!(internal_pk, priv_key_api._internal_key)
     }
 
+    #[test]
+    fn private_key_addition_under_p() -> Result<()> {
```
Comment (coltfred) [inline src/api_480.rs]: Test functions don't have to result in a `Result` type. if you take off the `-> Result<()>` you can also get rid of all the `Ok(())`

File: src/api.rs [hunk 2]
```diff
@@ -1085,6 +1086,22 @@ impl From<internal::PrivateKey<Monty256>> for PrivateKey {
     }
 }
 
+impl Add for PrivateKey {
+    type Output = Self;
+
+    fn add(self, other: Self) -> Self {
+        Self::from(self._internal_key + other._internal_key)
```
Comment (coltfred) [inline src/api.rs]: This is a style choice, but in this case I prefer `(self._internal_key + other._internal_key).into()`, which should infer the from instance that needs to be used. Feel free to ignore if you like this way better.

Comment (giarc3) [reply src/api.rs]: Done

File: src/api_480.rs (hunk 1 above)
Comment (giarc3) [reply src/api_480.rs]: Done

**Pull request merged**